config/jobs/kubernetes/sig-cloud-provider/azure/release-1.27.yaml

```diff
@@ -34,7 +34,7 @@ presubmits:
             - bash
             - -c
             - >-
-              cd ${GOPATH}/src/sigs.k8s.io/azuredisk-csi-driver && ./deploy/install-driver.sh master local,snapshot,enable-avset &&
```

Comment:
@andyzhangx is this diff expected between 1.23 and 1.27?

Reply:
from 1.24, csi driver install is required, so it should be 
```
cd ${GOPATH}/src/sigs.k8s.io/azuredisk-csi-driver && ./deploy/install-driver.sh master local,snapshot,enable-avset &&
```

Reply:
@andyzhangx this is all auto-generated from the `generate.sh` script... The script might need to get updated then?

Reply:
@nawazkh https://github.com/kubernetes/test-infra/pull/29307 should get you unblocked

Reply:
now that it merged, can you please rebase and run the generate script again?

Reply:
Updated! Thanks for the quick fix to generate.sh Cecile!